### DISCUSSION TITLE
Chemex fits look weird

### DISCUSSION THREAD

**Discussion by sucha1512** [gbouvignies/ChemEx#297] [Q&A] [answered]
Hi I am trying to analyse my trosy cpmg, and the fits look weird. can you let me know the problem? I have calculated errors from s to n ratio
[experiments.pdf](https://github.com/user-attachments/files/17230638/experiments.pdf)
Can you let me know what to do?

**Comment by viochemist** [comment, marked as answer]
Hello,

The pdf you show does not appear to have much (quantifiable) exchange,
although the R2's are very high so this must have been a real effort to
get! These types of fits typically result from poor initialization of
parameters. I find starting with simple models on an individual basis helps
to find good starting points and locate those residues with the most
likelihood of success. Then, when doing a global fitting, use the updated
values (R2, DeltaOmega, pb, kex) that you estimate from the initial fits
and use only those residues where the exchange appears significant. If you
are able to nail down reliable kex and pb values, then create a method with
those as fixed values and attempt to fit the rest of the residues.

Just my 2¢

Alex



*Dr Alexandar L Hansen <https://osu.academia.edu/AlexandarHansen>*
Lead Research Scientist
CCIC & Gateway NMR Facility
R224 CBEC, 151 W Woodruff Ave, Columbus, OH 43210
614.292.7133 Office / 716.341.8476 Mobile
***@***.*** ccic.ohio-state.edu
